Remotion/Core/Development.Analyzers/Core.Development.Analyzers.csproj

```diff
@@ -7,6 +7,7 @@
     <AssemblyName>Remotion.Development.Analyzers</AssemblyName>
     <AssemblyTitle>Remotion Analyzers</AssemblyTitle>
     <Description>Analyzer ensuring best practices when using the Remotion assemblies.</Description>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```

Comment:
What does the new property do? what happens by setting it true (or false)?

Reply:
As the name suggest it enforces analyzer best practices that are not enforced before. With the update the analyzer class had an error that complained that this property should be set. If we don't want to do all the work now we might be able to set this to `false`, which might disable the initial error as well.

Reply:
Good. 
@KuJo7 We should do a separate commit that fixes the trailing periods and introduces the build property and probably the unshipped.md file and document in the comment message that the change was required due to stronger enforcement of rules. Otherwise, we don't know why it was done when we look at the code history again. We accept that the build wouldn't work for this first revision without the changes since it's only in the feature branch. in theory, we could do the fix-commit first and then the upgrade-commit, but I don't think it's worth it.